src/bugsnag.js

```diff
@@ -188,6 +191,11 @@
     } else {
       crumb.name = truncate(crumb.name, 32);
       breadcrumbs.push(truncateDeep(crumb, 140));
+
+      // limit breadcrumb trail length, so the payload doesn't get too large
+      if (breadcrumbs.length > breadcrumbLimit) {
+        breadcrumbs = breadcrumbs.slice(0, breadcrumbLimit);
```

Comment:
This is correct. `breadcrumbs = breadcrumbs.slice(-1 * breadcrumbLimit)` would be the way to do it.